src/lib/mappingUtils.js

```diff
@@ -8,20 +8,30 @@ function getProcedureIntent(resource, resourceType) {
   )[0];
 }
 function getModalities(resource, resourceType) {
-  return fhirpath
-    .evaluate(
-      resource,
-      `${resourceType}.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique').extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality').valueCodeableConcept.coding.display`
-    )
-    .join("\n");
-}
-function getTechniques(resource, resourceType) {
-  return fhirpath
-    .evaluate(
-      resource,
-      `${resourceType}.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique').extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-technique').valueCodeableConcept.coding.display`
-    )
-    .join("\n");
+  const extensions = fhirpath.evaluate(
+    resource,
+    `${resourceType}.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique')`
+  );
+
+  return extensions.map((extension) => {
+    return {
+      Modality: extension.extension.find(
+        (extension) =>
+          extension.url ===
+          "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality"
+      ).valueCodeableConcept.coding[0].display,
```

Comment:
Do we need the `?` operator here like we added to a lot of other functions in this file?